src/Symfony/Component/Validator/Tests/Constraints/IbanValidatorTest.php

```diff
@@ -173,19 +181,25 @@ public function testInvalidIbans($iban, $code)
     public function getInvalidIbans()
     {
         return array(
-            array('CH93 0076 2011 6238 5295', Iban::CHECKSUM_FAILED_ERROR),
-            array('CH930076201162385295', Iban::CHECKSUM_FAILED_ERROR),
+            array('CH93 0076 2011 6238 5295', Iban::INVALID_FORMAT_ERROR),
+            array('CH930076201162385295', Iban::INVALID_FORMAT_ERROR),
+            array('NO93 8601117 947', Iban::INVALID_FORMAT_ERROR),
+            array('CH930072011623852957', Iban::INVALID_FORMAT_ERROR),
+            array('CY170020 128 0000 0012 0052 7600', Iban::INVALID_FORMAT_ERROR),
+            array('FR76', Iban::INVALID_FORMAT_ERROR),
```

Comment:
> I would like to see tests for

> valid IBANs from every country

@afurculita please add Brazil to the valid list, then all countries you are checking for have a valid test

```php
            array('BR9700360305000010009795493P1'), //Brazil
```

> invalid IBANs from every country

Bucket list:
AD, AE, AL, AT, AZ, BA, BE, BG, BH, BR, CR, CZ, DE, DO, DK, EE, ES, FI, FO, GB, GE, GI, GL, 
GR, GT, HR, HU, IE, IL, IS, IT, KW, KZ, LB, LI, LT, LU, LV, MC, MD, ME, MK, MR, MT, MU, NL, 
PK, PL, PS, PT, RO, RS, SA, SE, SI, SK, SM, TN, TR, VG,